src/components/BlogList/index.jsx

```diff
@@ -44,13 +50,26 @@ const BlogList = ({
         ))}
       </div>
       <div className="blog-list">
-        {filteredBlogsByTags.length
-          ? filteredBlogsByTags?.map((blog) => (
-              <BlogCard key={blog.id} blog={blog} />
-            ))
+        {currentBlogs?.length
+          ? currentBlogs?.map((blog) => <BlogCard key={blog.id} blog={blog} />)
           : "Blog not found"}
       </div>
-    </>
+      {(blogsPerPage < totalBlogs || blogsPerPage > totalBlogs) && (
```

Comment:
this button will always appear right? don't need to add condition for that. we can only add condition if there's no content

Reply:
updated